documentation/src/components/side-nav.ts

```diff
@@ -76,6 +76,7 @@ class SideNav extends LitElement {
                     <sp-sidenav variant="multilevel">
```

Comment:
I missed this, but being you are here, would you mind adding the `manage-tab-index` attribute to the `sp-sidenav`?